UPDATING.md

```diff
@@ -0,0 +1,115 @@
+# Updating Images
+The `docker-webkit-dev` project contains Docker images equipped with all the
+software needed to build and test WebKit. This document goes over how to
+maintain the images as new versions of the software are released.
+
+> [!NOTE]
+> This document assumes
+> [Docker Desktop](https://www.docker.com/products/docker-desktop/) is installed
+> and configured for Windows containers.
+
+## Finding new versions
+Software versions in Dockerfiles are manually determined. Check the software's
+site for updates using the :point_down: table.
+
+| Software | Download Link |
+|---|:---:|
+| git | https://github.com/git-for-windows/git/releases |
+| perl | https://strawberryperl.com/releases.html |
+| python | https://www.python.org/downloads |
+| pip | https://pypi.org/project/pip |
+| pywin32 | https://pypi.org/project/pywin32 |
+| ruby | https://rubyinstaller.org/downloads |
+| webrick | https://github.com/ruby/webrick/releases |
+| cmake | https://cmake.org/download |
+| ninja | https://ninja-build.org |
+| nuget | https://www.nuget.org/downloads |
+| llvm | https://github.com/llvm/llvm-project/releases |
+
+Every software in the Dockerfile has a corresponding `_VERSION` environment
+variable. Update this variable to install the newer version.
+
+Installing the software is done through the
+[WebKitDev PowerShell module](https://www.powershellgallery.com/packages/WebKitDev).
+If the download fails open an
+[issue](https://github.com/WebKitForWindows/powershell-webkit-dev/issues) there.
+
+> [!Warning]
+> The WebKitDev module expands a filename pattern for the installer. Any changes
+> to the naming scheme will cause download failures. The module may be replaced
+> by [Chocolatey](https://chocolatey.org) in the future.
+
+## Building Locally
+Use the `Build-All.ps1` PowerShell script to build the images. It expects a single argument `-Tag` which specifies the tag to build.
+
+The following tags are supported. For the latest information on Windows
+container version compatibility see the
+[documentation](https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility).
+
+| Tag Name  | Description |
+|---|---|
+| 2022 | A Windows 2022 server container |
+| windows-2022 | A Windows container, used for Layout Tests |
+| 1809 | A Windows 2019 server container |
+
```

Comment:
What situation should be concerned when a tag name needs to be specified? Is an appropriate tag defined depending on an OS such as Windows 10, Windows 11 etc?

Reply:
Hope I addressed this in https://github.com/WebKitForWindows/docker-webkit-dev/pull/367